src/api_server/swagger/firecracker.yaml

```diff
@@ -858,41 +858,47 @@ definitions:
     type: object
     required:
       - drive_id
-      - is_read_only
       - is_root_device
-      - path_on_host
     properties:
       drive_id:
         type: string
+      partuuid:
+        type: string
+        description:
+          Represents the unique id of the boot partition of this device. It is
+          optional and it will be taken into account only if the is_root_device
+          field is true.
+      is_root_device:
+        type: boolean
       cache_type:
         type: string
         description:
           Represents the caching strategy for the block device.
         enum: ["Unsafe", "Writeback"]
         default: "Unsafe"
+
+      # VirtIOBlock specific parameters
       is_read_only:
         type: boolean
-      is_root_device:
-        type: boolean
-      partuuid:
-        type: string
-        description:
-          Represents the unique id of the boot partition of this device. It is
-          optional and it will be taken into account only if the is_root_device
-          field is true.
+        description: Is block read only. This field is specific for virtio-block config.
       path_on_host:
         type: string
-        description: Host level path for the guest drive
+        description: Optional host level path for the guest drive. This field is specific for virtio-block config.
```

Comment:
why is it optional? It is required for virtio block, isn't it?

Reply:
Updated descriptions.